packages/flutter/lib/src/material/dialog.dart

```diff
@@ -457,13 +457,18 @@ class _DialogRoute<T> extends PopupRoute<T> {
 
 /// Displays a dialog above the current contents of the app.
 ///
-/// This function typically receives a [Dialog] widget as its child argument.
-/// Content below the dialog is dimmed with a [ModalBarrier].
+/// This function typically receives a [Dialog] widget inside of a [WidgetBuilder]
```

Comment:
not sure what you mean by "inside of a WidgetBuilder".

Reply:
Rephrased